src/PodActions.jsx

```diff
@@ -89,7 +89,7 @@ export const PodActions = ({ onAddNotification, pod }) => {
                                   client.postPod(pod.isSystem, "stop", pod.Id, { t: 0 })
                                           .catch(ex => {
                                               const error = cockpit.format(_("Failed to force stop pod $0"), pod.Name);
-                                              this.props.onAddNotification({ type: 'danger', error, errorDetail: ex.message });
+                                              onAddNotification({ type: 'danger', error, errorDetail: ex.message });
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1208-20230217-061044-8d371074-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)